### PR TITLE
docs: update member list

### LINKED ISSUE #499: Membership audit - please respond
We are doing a membership audit as suggested in https://github.com/nodejs/package-maintenance/issues/497.  Unless you have been in a recent meeting or respond to this issue within the next 30 days, we'll assume it's ok to move you to emeritus.

Michael Dawson (@mhdawson)
Matteo Collina (@mcollina)
Christopher Hiller (@boneskull)
Matt Edelman (@grawk)
Franziska Hinkelmann (@fhinkel)
Refael Ackermann (@refack)
Jordan Harband (@ljharb)
Charlie Robbins (@indexzero)
Monica EnePietrosanu (@menepie)
Arunesh Chandra (@aruneshchandra)
Mohammed Essehemy (@MohammedEssehemy)
Emily Platzer (@craftninja)
Viktor Vershanskij (@wentout)
Kyle Martin (@js-kyle)
Wes Todd (@wesleytodd)
Pooya Parsa (@pi0)
Pavel Vostrikov (@vostrik)
Larry Lu (@Larry850806)
Osmond van Hemert (@ovhemert)
Nairi Harutyunyan (@nairihar)
Edward Snare (@edwardsnare)
Joel Chen (@jchip)
Kenigbolo Meya Stephen (@kenigbolo)
Victor Alencar (@valencar)
Waqar Mohammad (@waqardm)
Lian Duan (@dlhandsome)
Chan Gong (@Gcaufy)
Xin Du (@clarkdo)
Alexey Pyltsyn (@lex111)
Manuel Spigolon (@eomm)
Zied CHOUK (@ZiedCHOUK)
James Youngs (@sidhuko)
Rob Simpson (@pertrai1)
Amelia Garripoli (@argoc)
Esphas Kueen (@esphas)
Antonio Gonzalez (@bluesockets)
Satyam Bansal(@satyam22)
Ankur Oberoi (@aoberoi)
Felix-Antoine Paradis (@reel)
Glenn Hinks (@ghinks)
Abdullah Tariq (@abdullahtariq91)
Mike Perry Y Attara (@mikeattara)
Kamal Singh Rawat (@ksrawat)
Dominykas Blyžė (@dominykas)
Jose Navarro (@jmn8718)
Keith Holliday (@thehollidayinn)
Ian Sutherland (@iansu)
Lars Graubner (@lgraubner)
Chandan Kumar (@chandanrjit)
Christian Kaatz (@chrkaatz)
Arne Schubert (@atd-schubert)
Ravi Joshi (@Jo-ravar)
Benjamin Chen (@BeniCheni)
Pothamsetty K (@potham)
Yash Handa (@Yash-Handa)
Ke Yang (@aojiaotage)
Tamil Vendhan Kanagarasu (@mudivili)
Gentian Elmazi (@gentios)
Kamesh Dashora (@kamesh95)
Alessio Dionisi (@adnsio)
Zeke Sikelianos (@zek)
Rick Markins (@rxmarbles)
Shailesh Kanzariya(@shailesh-kanzariya)
Hamza Baig (@hamxabaig)
Saif Eddine Jerbi (@JerbiSaif)
Edgar Muentes (@emuentes)
Karan Vohra (@kvohra3)
Tommy York (@tommyyork)
Wilbert Abreu (@wilbert-abreu)
Terary Chambers (@terary)
Konstantin Bubyakin (@Goodluckhf)
Alexander Kolesnikov (@eddort)
Anatoly Volgin (@zeroWheel)
Karan Sapolia (@karansapolia)
Andrew Kuzmenko (@andrewkuzmenko)
Zubov Kirill (@Mobydack)
Petr Grishin (@petrgrishin)
Murukesh Sadasivan (@murukesh)
Frank Lemanschik (@frank-dspeed)
Nardyuzhev Vadim (@vadinN)
James George (@jamesgeorge007)
Gabriel Schulhof (@gabrielschulhof)
Minh Nguyen (@truongminh)
Jacopo Daeli (@JacopoDaeli)
Hassan Sani (@inidaname)
Ahmad Abdul-Aziz (@devamaz)
Samuel Omole (@samie820)
Sathish Gajendran (@sathishgajendran)
Abdul Rahman Zantout (@abedzantout)
James M. Greene (@JamesMGreene)
Bethany Griggs (@BethGriggs)
Mayank Patel (@mayank-patel)
Lucas Holmquist (@lholmquist)
Darcy Clarke (@darcyclarke)
Giorgos Ntemiris (@gntem)
Helio Frota (@helio-frota)
Steven (@styfle)
Owen Buckley (@thescientist13)
Pranshu Srivastava (@rexagod)
Larson Carter (@larson-carter)
Jon Church (@jonchurch)
Antoni Kepinski (@xxczaki)
Rodion Abdurakhimov (@rodion-arr)
Ella Polo (@polec1)
Ash Cripps (@AshCripps)
Sachin Jangid (@gittysachin)
Zack Schuster (@zackschuster)

### PR DESCRIPTION
**Pull request by Eomm**
Closes #499

### CODE REVIEW

File: EMERITUS.md [hunk 1]
```diff
@@ -2,3 +2,74 @@
...
+* Murukesh Sadasivan ([@murukesh](https://github.com/murukesh))
+* James George ([@jamesgeorge007](https://www.github.com/jamesgeorge007))
+* Gabriel Schulhof ([@gabrielschulhof](https://www.github.com/gabrielschulhof))
+* Minh Nguyen ([@truongminh](https://github.com/truongminh))
+* Jacopo Daeli ([@JacopoDaeli](https://github.com/JacopoDaeli))
+* Ahmad Abdul-Aziz ([@devamaz](https://github.com/devamaz))
+* Samuel Omole ([@samie820](https://github.com/samie820))
+* Abdul Rahman Zantout ([@abedzantout](https://github.com/abedzantout))
+* James M. Greene ([@JamesMGreene](https://github.com/JamesMGreene))
+* Bethany Griggs ([@BethGriggs](https://github.com/bethgriggs))
```
[the remark this answers is not in the document]
Comment (BethGriggs) [reply EMERITUS.md:68]: Could I be added back please? Missed the pings and had a lapse in activity, but there are definitely some upcoming discussions I'd like to be involved in

File: MEMBERS.md [hunk 2]
```diff
@@ -1,109 +1,38 @@
...
-* Jacopo Daeli ([@JacopoDaeli](https://github.com/JacopoDaeli))
 * Hassan Sani ([@inidaname](https://github.com/inidaname))
-* Ahmad Abdul-Aziz ([@devamaz](https://github.com/devamaz))
-* Samuel Omole ([@samie820](https://github.com/samie820))
 * Sathish Gajendran ([@sathishgajendran](https://github.com/sathishgajendran))
-* Abdul Rahman Zantout ([@abedzantout](https://github.com/abedzantout))
-* James M. Greene ([@JamesMGreene](https://github.com/JamesMGreene))
-* Bethany Griggs ([@BethGriggs](https://github.com/bethgriggs))
-* Mayank Patel ([@mayank-patel](https://github.com/mayank-patel))
 * Lucas Holmquist ([@lholmquist](https://github.com/lholmquist))
```
Comment (ljharb) [inline MEMBERS.md:30]:
```suggestion
* Bethany Griggs ([@BethGriggs](https://github.com/bethgriggs))
* Lucas Holmquist ([@lholmquist](https://github.com/lholmquist))
```

**Review by thescientist13 — APPROVED** [review]

**Comment by ljharb** [comment]
@mhdawson if we create github teams for these (happy to volunteer to do so) then could we use ncu-team-sync to keep this list updated?

File: EMERITUS.md [hunk 3]
```diff
@@ -2,3 +2,73 @@
...
+* Jacopo Daeli ([@JacopoDaeli](https://github.com/JacopoDaeli))
+* Ahmad Abdul-Aziz ([@devamaz](https://github.com/devamaz))
+* Samuel Omole ([@samie820](https://github.com/samie820))
+* Abdul Rahman Zantout ([@abedzantout](https://github.com/abedzantout))
+* James M. Greene ([@JamesMGreene](https://github.com/JamesMGreene))
+* Mayank Patel ([@mayank-patel](https://github.com/mayank-patel))
+* Giorgos Ntemiris ([@gntem](https://github.com/gntem))
+* Helio Frota ([@helio-frota](https://github.com/helio-frota))
+* Pranshu Srivastava ([@rexagod](https://github.com/rexagod))
+* Rodion Abdurakhimov ([@rodion-arr](https://github.com/rodion-arr))
```
Comment (rodion-arr) [inline EMERITUS.md:72]: I'd like to stay in members list as I replied in comment to #499

**Review by rxmarbles — APPROVED** [review]

**Review by mhdawson — APPROVED** [review]
LGTM

**Comment by Eomm** [comment]
> @mhdawson if we create github teams for these (happy to volunteer to do so) then could we use ncu-team-sync to keep this list updated?

It seems a great idea to me 👍🏽

**Comment by bnb** [comment]
Since we're updating the member list, I should be added? I'm not sure why I'm missing from both lists - IIRC I joined early and have contributed to the various projects under package maintenance consistently (albeit not in meetings as often as I'd like). I'm still in the package maintenance team on GitHub.

File: EMERITUS.md [hunk 4]
```diff
@@ -2,3 +2,72 @@
...
+* Ahmad Abdul-Aziz ([@devamaz](https://github.com/devamaz))
+* Samuel Omole ([@samie820](https://github.com/samie820))
+* Abdul Rahman Zantout ([@abedzantout](https://github.com/abedzantout))
+* James M. Greene ([@JamesMGreene](https://github.com/JamesMGreene))
+* Mayank Patel ([@mayank-patel](https://github.com/mayank-patel))
+* Giorgos Ntemiris ([@gntem](https://github.com/gntem))
+* Helio Frota ([@helio-frota](https://github.com/helio-frota))
+* Pranshu Srivastava ([@rexagod](https://github.com/rexagod))
+* Ella Polo ([@poloec1](https://github.com/poloec1))
+* Ash Cripps ([@AshCripps](https://github.com/AshCripps))
```
Comment (ljharb) [inline EMERITUS.md:73]:
```suggestion
* Ash Cripps ([@AshCripps](https://github.com/AshCripps))
* Tierney Cyren ([@bnb](https://github.com/bnb)]
```

this list is manual; when we DO use ncu-team-sync, it'd be automated

File: MEMBERS.md [hunk 5]
```diff
@@ -1,109 +1,40 @@
...
 * Owen Buckley ([@thescientist13](https://github.com/thescientist13))
-* Pranshu Srivastava ([@rexagod](https://github.com/rexagod))
 * Larson Carter ([@larson-carter](https://github.com/larson-carter))
 * Jon Church ([@jonchurch](https://github.com/jonchurch))
 * Antoni Kepinski ([@xxczaki](https://github.com/xxczaki))
 * Rodion Abdurakhimov ([@rodion-arr](https://github.com/rodion-arr))
-* Ella Polo ([@polec1](https://github.com/poloec1))
-* Ash Cripps ([@AshCripps](https://github.com/AshCripps))
 * Sachin Jangid ([@gittysachin](https://github.com/gittysachin))
 * Zack Schuster ([@zackschuster](https://github.com/zackschuster))
```
Comment (ljharb) [inline MEMBERS.md:40]:
```suggestion
* Zack Schuster ([@zackschuster](https://github.com/zackschuster))
* Tierney Cyren ([@bnb](https://github.com/bnb)]
```

this list is manual; when we DO use ncu-team-sync, it'd be automated

File: MEMBERS.md [hunk 6]
```diff
@@ -1,109 +1,41 @@
...
 * Esphas Kueen ([@esphas](https://github.com/esphas))
-* Antonio Gonzalez ([@bluesockets](https://github.com/bluesockets))
-* Satyam Bansal([@satyam22](https://github.com/satyam22))
-* Ankur Oberoi ([@aoberoi](https://github.com/aoberoi))
 * Felix-Antoine Paradis ([@reel](https://github.com/reel))
-* Glenn Hinks ([@ghinks](https://github.com/ghinks))
-* Abdullah Tariq ([@abdullahtariq91](https://github.com/abdullahtariq91))
 * Mike Perry Y Attara ([@mikeattara](https://github.com/mikeattara))
-* Kamal Singh Rawat ([@ksrawat](https://github.com/ksrawat))
-* Dominykas Blyžė ([@dominykas](https://github.com/dominykas))
```
Comment (dominykas) [inline MEMBERS.md:46]: Hey!

File: MEMBERS.md [hunk 7]
```diff
@@ -1,109 +1,41 @@
...
 * Zied CHOUK ([@ZiedCHOUK](https://github.com/ZiedCHOUK))
-* James Youngs ([@sidhuko](https://github.com/sidhuko))
-* Rob Simpson ([@pertrai1](https://github.com/pertrai1))
-* Amelia Garripoli ([@argoc](https://github.com/argoc))
 * Esphas Kueen ([@esphas](https://github.com/esphas))
-* Antonio Gonzalez ([@bluesockets](https://github.com/bluesockets))
-* Satyam Bansal([@satyam22](https://github.com/satyam22))
-* Ankur Oberoi ([@aoberoi](https://github.com/aoberoi))
 * Felix-Antoine Paradis ([@reel](https://github.com/reel))
-* Glenn Hinks ([@ghinks](https://github.com/ghinks))
```
Comment (dominykas) [inline MEMBERS.md:42]: I think Glenn is still active?

Comment (Eomm) [reply MEMBERS.md:42]: I must say that I automated it a bit 😄 

@ghinks could you confirm?

Comment (mhdawson) [reply MEMBERS.md:42]: +1, Glen has been active.

**Review by mhdawson — APPROVED** [review]
LGTM with Glenn left as active.

File: MEMBERS.md (hunk 7 above)
Comment (Eomm) [reply MEMBERS.md:42]: done 👍🏽

**Comment by bnb** [comment]
Also closes #497 AFAIK

**Comment by mhdawson** [comment]
@Eomm are you still going to be able to make the last updates and then land this.

**Review by zackschuster — APPROVED** [review]

**Comment by Eomm** [comment]
> 

I think all the comments have been addressed. Am I missing something?

**Review by ljharb — APPROVED** [review]
seems good to land to me

**Comment by mhdawson** [comment]
> I think all the comments have been addressed. Am I missing something?

Landing

**Pull request merged**